tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -240,6 +240,20 @@ public function userSendsTheFollowingShareInvitationUsingTheGraphApi(string $use
 		);
 	}
 
+	/**
+	 * @When user :user updates the permissions of last share using the Graph API:
+	 *
+	 * @param string $user
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 */
+	public function userUpdatesThePermissions($user, TableNode $table) {
+		$response = $this->featureContext->shareNgGetLastCreatedUserGroupShare();
+		$shareID = json_decode($response->getBody()->getContents())->value[0]->id;
+		$this->updateLinkShare($user, $table, $shareID);
```

Comment:
`updateLinkShare` is not suitable here
```suggestion
		$this->updateShare($user, $table, $shareID);
```